tests/test-integration/src/test/java/org/eclipse/jetty/test/HttpInputIntegrationTest.java

```diff
@@ -91,8 +92,10 @@ public static void beforeClass() throws Exception
 
         // SSL Context Factory for HTTPS and HTTP/2
         __sslContextFactory = new SslContextFactory.Server();
-        __sslContextFactory.setKeyStorePath("src/test/resources/keystore.p12");
-        __sslContextFactory.setKeyStorePassword("storepwd");
+        __sslContextFactory.setKeyStore(TestKeyStoreFactory.getServerKeyStore());
+        __sslContextFactory.setKeyStorePassword(TestKeyStoreFactory.KEY_STORE_PASSWORD);
+        __sslContextFactory.setTrustStore(TestKeyStoreFactory.getTrustStore());
+        __sslContextFactory.setTrustStorePassword(TestKeyStoreFactory.KEY_STORE_PASSWORD);
```

Comment:
Don't change the purpose and scope of the test.
This SslContextFactory didn't have a TrustStore for a reason.